src/FakeItEasy/FakeItEasy.csproj

```diff
@@ -280,7 +280,10 @@
   <ItemGroup />
   <ItemGroup>
     <EmbeddedResource Include="Creation\CastleDynamicProxy\DynamicProxyResources.resx" />
-    <EmbeddedResource Include="ExceptionMessages.resx" />
+    <EmbeddedResource Include="ExceptionMessages.resx">
+      <Generator>PublicResXFileCodeGenerator</Generator>
+      <LastGenOutput>ExceptionMessages1.Designer.cs</LastGenOutput>
+    </EmbeddedResource>

```

Comment:
The IDE tried to use this by default. I don't see any setup for generating that file, it seems to be disconnected and commited as a regular source file in the project.